bedrock/foundation/templates/foundation/trademarks/list.html

```diff
@@ -9,124 +9,242 @@
     </tr>
     <tr>
-      <td><img width="100" height="100" alt="{{ _('Bugzilla logo') }}™" src="{{ static('img/trademarks/bugzilla-tm.png') }}" /><span>&trade;</span></td>
-      <th>{{ _('Bugzilla logo') }}</th>
+      <td><img width="100" height="100" alt="{{ _('Pontoon logo') }}™" src="{{ static('img/trademarks/pontoon-logo.svg') }}" /></td>
+      <th>{{ _('Pontoon') }}</th>
     </tr>
     <tr>
-      <td><img width="100" height="100" alt="{{ _('Sunbird logo') }}®" src="{{ static('img/trademarks/sunbird-logo-r.png') }}" /><span>&reg;</span></td>
-      <th>{{ _('Sunbird logo') }}</th>
+      <td><img width="100" height="100" alt="{{ _('QMO logo') }}™" src="{{ static('img/trademarks/qmo-logo-tm.png') }}" /></td>
+      <th>{{ _('QMO') }}</th>
     </tr>
     <tr>
-      <td><img width="100" height="100" alt="{{ _('SeaMonkey logo') }}®" src="{{ static('img/trademarks/seamonkey-logo-r.png') }}" /><span>&reg;</span></td>
-      <th>{{ _('SeaMonkey logo') }}</th>
+      <td><img width="100" height="100" alt="{{ _('Rust logo') }}™" src="{{ static('img/trademarks/rust-logo.png') }}" /></td>
+      <th>{{ _('Rust') }}</th>
     </tr>
     <tr>
-      <td><img width="100" height="100" alt="{{ _('Foxkeh logo') }}®" src="{{ static('img/trademarks/foxkeh-logo-r.png') }}" /><span>&reg;</span></td>
-      <th>{{ _('Foxkeh logo') }}<br /><small>{{ _('This trademark is licensed under the Creative Commons Attribution-Non-Commercial-NoDerivs 2.1 Japan.') }}</small></th>
+      <td><img width="100" height="100" alt="{{ _('SeaMonkey logo') }}®" src="{{ static('img/trademarks/seamonkey-logo-r.png') }}" /></td>
+      <th>{{ _('SeaMonkey') }}</th>
     </tr>
     <tr>
-      <td><img width="100" height="100" alt="{{ _('Creative Collective logo') }}™" src="{{ static('img/trademarks/creativecollective-logo.png') }}" /><span>&trade;</span></td>
-      <th>{{ _('Creative Collective logo') }}</th>
+      <td><img width="100" height="100" alt="{{ _('SUMO logo') }}™" src="{{ static('img/trademarks/sumo-logo.png') }}" /></td>
+      <th>{{ _('SUMO') }}</th>
     </tr>
     <tr>
-      <td><img width="100" height="100" alt="{{ _('Campus Reps logo') }}™" src="{{ static('img/trademarks/campusreps-logo.png') }}" /><span>&trade;</span></td>
-      <th>{{ _('Campus Reps logo') }}</th>
+      <td><img width="100" height="100" alt="{{ _('Sunbird logo') }}®" src="{{ static('img/trademarks/sunbird-logo-r.png') }}" /></td>
+      <th>{{ _('Sunbird') }}</th>
     </tr>
     <tr>
-      <td><img width="100" height="100" alt="{{ _('Rust logo') }}™" src="{{ static('img/trademarks/rust-logo.png') }}" /><span>&trade;</span></td>
-      <th>{{ _('Rust logo') }}</th>
+      <td><img width="100" height="100" alt="{{ _('Thunderbird logo') }}®" src="{{ static('img/trademarks/thunderbird_logo_100.png') }}" /></td>
+      <th>{{ _('Thunderbird') }}</th>
     </tr>
+  </table>
+
+  <h2>{{ _('Pocket Trademark List') }}</h2>
+
+  <h3>{{ _('Wordmarks') }}</h3>
+
+  <ul id="marks">
```

Comment:
Nit: we should avoid duplicate IDs. Not technically breaking anything or even invalid, it's just not really correct. If this is being used for styling we should change to a class so it's more reusable.